htdocs/debugbar/class/DebugBar.php

```diff
@@ -40,7 +40,7 @@ public function __construct()
 		//$this->addCollector(new DolExceptionsCollector());
 		$this->addCollector(new DolQueryCollector());
 		$this->addCollector(new DolibarrCollector());
-		if ($conf->syslog->enabled) {
+		if (!empty($conf->syslog->enabled)) {
```

Comment:
you can use isModEnabled('syslog')

Reply:
isModEnabled is in v16 ? I thought it was in v17... Thanks @frederic34, PR modified